README.md

```diff
@@ -201,6 +207,95 @@ const rule = {
 };
 ```
 
+### Includes
```

Comment:
One thing that might be confusing to a new reader is the following, what exactly is `input` for each example?

Reply:
:+1:, i'll add more details [here](https://github.com/godaddy/node-flipr#flipr-rules)